app/src/main/java/org/oppia/android/app/profile/ResetPinDialogFragmentPresenter.kt

```diff
@@ -68,7 +68,7 @@ class ResetPinDialogFragmentPresenter @Inject constructor(
     val dialog = AlertDialog.Builder(activity, R.style.OppiaAlertDialogTheme)
       .setTitle(R.string.reset_pin_enter)
       .setView(binding.root)
-      .setMessage("This PIN wil be $name's new PIN and will be required when signing in.")
+      .setMessage("This PIN will be $name's new PIN and will be required when signing in.")
```

Comment:
This actually should be an app string so that it can be localized. I'm a bit surprised we missed this.

Could you move this to strings.xml? You can see other strings that take variables for a reference on how to pass a value to a localized string.

Also, please add a test to verify that the dialog's message is correct when opened.